dolphinscheduler-dao/src/main/resources/sql/upgrade/2.0.0_schema/mysql/dolphinscheduler_ddl.sql

```diff
@@ -415,7 +415,7 @@ alter table t_ds_process_instance drop dependence_schedule_times;
 
 -- t_ds_task_instance   note: Data migration is not supported
 alter table t_ds_task_instance change process_definition_id task_code bigint(20) NOT NULL COMMENT 'task definition code';
-alter table t_ds_task_instance add task_definition_version int(11) DEFAULT '0' COMMENT 'task definition version' AFTER task_code;
+alter table t_ds_task_instance add task_definition_version int(11) DEFAULT '1' COMMENT 'task definition version' AFTER task_code;
```

Comment:
The default value of t_ds_task_instance is '0', we cannot change this.

Reply:
Or I could add it here ？
`update t_ds_task_instance set task_definition_version ='1';`

Reply:
LGTM only update the `task_definition_version`, but the version in t_ds_task_instance should start from `0` this is strange.

Reply:
t_ds_task_inset.task_definition_version is assigned to 1 for the first time, which is not useful when the default value is 0,the data in the table should not have t_ds_task_instance task_definition_version = 0,so `update t_ds_task_instance set task_definition_version ='1'`

Reply:
Oh, I see the code will set the begin version to 0. So set the default version to 1 is the correct way.
Could you please change the default value in the initialize sql.
https://github.com/apache/dolphinscheduler/tree/dev/dolphinscheduler-dao/src/main/resources/sql
And we also need to alter this latest version https://github.com/apache/dolphinscheduler/tree/dev/dolphinscheduler-dao/src/main/resources/sql/upgrade/3.2.1_schema.